### DISCUSSION TITLE
Broken lib links?

### DISCUSSION THREAD

**Discussion by antonioleal** [pawelsalawa/sqlitestudio#4496] [Q&A] [answered]
Hi, after building on my slack box I notice the following broken links in the app's lib folder

libk5crypto.so.3 -> libk5crypto.so.3.1
libk5crypto.so.3.1 -> ../../lib64/libk5crypto.so.3.1
libkrb5.so.3 -> libkrb5.so.3.3
libkrb5.so.3.3 -> ../../lib64/libkrb5.so.3.3
libkrb5support.so.0 -> libkrb5support.so.0.1
ibkrb5support.so.0.1 -> ../../lib64/libkrb5support.so.0.1

are these needed ?

Antonio

**Comment by pawelsalawa** [comment, marked as answer]
k5crypto and krb5 are dependencies introduced by Qt. The answer to why are they pointing to `../../*` I do not know. When building on Ubuntu, these dependencies are resolved to their absolute paths (i.e. `libkrb5.so.3 => /lib/x86_64-linux-gnu/libkrb5.so.3`). This depends on how particular binary was compiled (linked, to be precise).

**Reply by antonioleal** [reply]
Hi,

I've build using the sources scripts from /scripts/linux

basically extract source downloaded from github

and running

./compile.sh /usr/lib64/qt5/bin/qmake 4
./create_portable.sh /usr/lib64/qt5/bin/qmake

Did not try on Ubuntu
